public/app/features/explore/utils/ResultProcessor.test.ts

```diff
@@ -206,160 +201,4 @@ describe('ResultProcessor', () => {
       });
     });
   });
-
-  describe('constructed with result that is a DataQueryResponse and merging with previous results', () => {
```

Comment:
Not needed as the mode when ResultProcessor was buffering the logs data itself is not there any more.